### PR TITLE
Update Portuguese with missing translations

### PR DESCRIPTION
**Pull request by FaahdSolano**
# Changes

Added some portuguese translation (file src/nls/messages_pt.json) as described in issue #3068

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (new features, significant UI changes, API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by AlanGreene** [comment]
Thanks for the pR @FaahdSolano 
/ok-to-test

File: src/nls/messages_pt.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+  "dashboard.about.description": "Tekton é um poderoso e flexível framework open-source para criação de sistemas CI/CD, permitindo que desenvolvedores possam construir, testar, e implantar através de provedores de nuvem e sistemas on-premises",
+  "dashboard.about.documentation": "Documentação e recursos",
+  "dashboard.about.environmentDetails": "Detalhes de ambiente",
+  "dashboard.about.error": "Erro ao receber dados",
   "dashboard.about.isReadOnly": "",
   "dashboard.about.logoutURL": "",
-  "dashboard.about.missingProperties": "",
-  "dashboard.about.title": "",
+  "dashboard.about.missingProperties": "Não foi possível encontrar: {errorsFound}",
+  "dashboard.about.title": "Sobre Tekton",
```
Comment (AlanGreene) [inline src/nls/messages_pt.json:24]:
```suggestion
  "dashboard.about.title": "Sobre Tekton",
```

Can you remove the trailing space at the end of this line? That should resolve the issue with the tests.

**Review by AlanGreene — CHANGES_REQUESTED** [review]

**Review by AlanGreene — CHANGES_REQUESTED** [review]
Looks like there's a trailing newline at the end of the file. Can you remove that and squash your commits please? Otherwise looks good, thanks for the translations.

**Comment by FaahdSolano** [comment]
Thanks for the patience, first time contributing

**Comment by AlanGreene** [comment]
> Thanks for the patience, first time contributing

No worries 😸 

Looks like the newline at the end is still there. Perhaps your editor is adding it automatically? If you're using an editor with support for [EditorConfig](https://editorconfig.org/#pre-installed), or that supports plugins providing this functionality, we have this set https://github.com/tektoncd/dashboard/blob/b766740b3f2fd91b3665973d193110062ea77941/.editorconfig#L18

Can you also squash your commits so there's just 1 commit in the PR? See the contributing guide: https://github.com/tektoncd/community/blob/main/standards.md#small-pull-requests
This helps us to maintain a clean and informative commit history.

**Comment by AlanGreene** [comment]
The integration tests failure is unrelated to your change and should be resolved now
/test pull-tekton-dashboard-integration-tests

**Comment by AlanGreene** [comment]
Thanks again for the translations @FaahdSolano. I've opened https://github.com/tektoncd/dashboard/pull/3098 which includes your changes and resolves the issue with the trailing newline.

**Pull request closed without merging**